.github/workflows/labeler.yml

```diff
@@ -26,9 +29,7 @@ jobs:
             "members": "read"
           }
     - name: Add agent-python label
-      uses: actions-ecosystem/action-add-labels@v1
-      with:
-        labels: agent-python
+      run: gh issue edit "$NUMBER" --add-label "agent-python"
```

Comment:
@v1v This is not working, NUMBER is empty. See https://github.com/elastic/apm-agent-python/actions/runs/14264122101/job/39982015773?pr=2261

Reply:
Uhm, I'll review it